applications/openshift/authentication/oauth_or_oauthclient_inactivity_timeout/policy/stig/shared.yml

```diff
@@ -0,0 +1,29 @@
+checktext: |-
+  To check if the OAuth server timeout is configured, run the following command:
```

Comment:
These three lines look inconsistent with the draft. Are they duplicate?

Reply:
Nice catch, I used the wrong rule and based the reply on the rule's OCIL. Right now the draft mandates that the client timeout is set which is different from the NIST profiles that mandate either client or server.
I changed the patch to be in sync with the draft, but I also filed a card in our Jira to follow up with DISA because I think that the server side timeout is fine, too and less brittle and possible to remediatie automatically.